test/validation-autocompleter.test.js

```diff
@@ -0,0 +1,236 @@
+const { EditSession } = require('brace');
+const ace = require('brace');
+const { ValidationAutoCompleter, QUERY_OPERATORS } = require('../');
+
+require('brace/mode/javascript');
+require('brace/ext/language_tools');
+
+const { Mode } = ace.acequire('ace/mode/javascript');
+const { textCompleter } = ace.acequire('ace/ext/language_tools');
+
+describe('ValidationAutoCompleter', () => {
+  const fields = [
+    {
+      name: 'name',
+      value: 'name',
+      score: 1,
+      meta: 'field',
+      version: '0.0.0'
```

Comment:
```suggestion
      version: '3.6.0'
```
To fix test failures

Reply:
Hah just pushed and saw your comment. You are so fast in reviewing!